repositories/meteorsRepository.js

```diff
@@ -0,0 +1,13 @@
+const axios = require("axios");
+
+const getMeteorsInfo = async (startDate, endDate) => {
+  const nasaToken = process.env.NASA_API_KEY;
+  const nasaApiUrl = process.env.NASA_API_URL;
```

Comment:
create please config file and move env variables https://teams.microsoft.com/l/message/19:6a563893960e4d50a159c3859a776a7d@thread.v2/1728559851120?context=%7B%22contextType%22%3A%22chat%22%7D